internal/locationutil/location.go

```diff
@@ -0,0 +1,52 @@
+package locationutil
+
+import (
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/values"
+)
+
+type Location struct {
+	Name   string
+	Offset flux.Duration
+}
+
+func (l Location) IsUTC() bool {
+	name := l.Name
+	if name == "" {
+		name = "UTC"
+	}
+	return name == "UTC" && l.Offset.IsZero()
+}
+
+func (l Location) Load() (LocationZone, error) {
+	name := l.Name
+	if name == "" {
+		name = "UTC"
+	}
+	loc, err := LoadLocation(name)
+	if err != nil {
+		return LocationZone{}, err
+	}
+	loc.Offset = l.Offset
+	return loc, nil
+}
+
+func (l Location) ToUTCTime(t values.Time) values.Value {
+	// load location
+	LocationZone, err := l.Load()
```

Comment:
It is really important we do not throw away this error. I think I get that this code is meant to be reused within the date package, but that means this code should be inside of the date package rather than a util package. Under no scenario should it ever throw away the error and return a valid value.

This function also doesn't follow the typical Go naming conventions. Local variables are intended to be named with `mixedCaps`. This helps differentiate them from exported functions within the same package.